File: app/src/main/java/com/example/artur/watch/Model/Capitulo.java
package com.example.artur.watch.Model;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.relation.ToOne;

@Entity
public class Capitulo {

    @Id  private long id;
    private String titulo;
    private String descricao;
    private ToOne<Temporada> temporada;

    public Capitulo() { }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public ToOne<Temporada> getTemporada() {
        return temporada;
    }

    public void setTemporada(ToOne<Temporada> temporada) {
        this.temporada = temporada;
    }
}

File: app/src/main/java/com/example/artur/watch/Model/Capitulo.kt
package com.example.artur.watch.Model

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import io.objectbox.relation.ToOne

@Entity
class Capitulo {

    constructor()

    constructor(titulo: String, descricao: String){
        this.titulo = titulo
        this.descricao = descricao
    }

    @Id
    var id: Long = 0
    lateinit var titulo: String
    lateinit var descricao: String
    var nTemporada: Int = 0
    var nCapitulo: Int = 0
    lateinit var serie: ToOne<Serie>

    override fun toString(): String {
        return if (serie.target.tipo == "Filme") this.descricao else
            "T${this.nTemporada}: E${this.nCapitulo}\n${this.descricao}"
    }
}
